docs/index.md

```diff
@@ -28,7 +28,7 @@ This tool is intended for use by developers who are experienced with JavaScript.
 ## Including the Buy SDK
 
 ```html
-<script src="http://sdks.shopifycdn.com/js-buy-sdk/latest/shopify-buy.polyfilled.globals.min.js"></script>
+<script src="http://sdks.shopifycdn.com/js-buy-sdk/v0/latest/shopify-buy.umd.polyfilled.min.js"></script>
```

Comment:
For "best" CDN distributions I think it's best to look at how other projects handle this.

https://code.jquery.com/

They basically do the same thing where they have a major version number they distribute.

But... Since all versions live at the root I'm not 100% sure we need `latest/` in the url.

Could be just:
```
v0/shopify-buy.umd.js
v0.3.0/shopify-buy.umd.js
v0.3.1/shopify-buy.umd.js
v1/shopify-buy.umd.js
v1.0.0/shopify-buy.umd.js
v1.1.0/shopify-buy.umd.js
```